tutorial/01_Introduction/intro.ipynb

```diff
@@ -8,7 +8,7 @@
    "source": [
```

Comment:
<p>see comment above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hpc4cmb/toast/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='87'/>

Reply:
<p>I was inspired by the traitlets methods <code>traits()</code>  and <code>class_traits()</code>,  but I can add a "get_" in there if it is more clear.</p>

Reply:
yes please